app/src/main/java/pl/edu/zut/mad/appzut/activities/MainActivity.java

```diff
@@ -3,54 +3,74 @@
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
 import android.support.v7.app.AppCompatActivity;
-import android.view.Menu;
-import android.view.MenuItem;
 
 import pl.edu.zut.mad.appzut.R;
 import pl.edu.zut.mad.appzut.fragments.CalendarFragment;
 import pl.edu.zut.mad.appzut.fragments.ScheduleFragment;
 
 public class MainActivity extends AppCompatActivity {
 
+    private static final String CALENDAR_TAG = "calendar_fragment";
+    private static final String SCHEDULE_TAG = "schedule_fragment";
+    private CalendarFragment calendarFragment = new CalendarFragment();
+    private ScheduleFragment scheduleFragment = new ScheduleFragment();
```

Comment:
When activity state is restored fragment instances created here won't be used, this might hide bugs in the future.